ONSdigital/eq-author-app
# EAR 1633 remove editor privilege

JIRA ticket: https://collaborate2.ons.gov.uk/jira/browse/EAR-1633

### What is the context of this PR?

- You can remove yourself as an editor in a Questionnaire, but it does not inform you that you can no longer edit it via the red heading bar.

### How to review

1. Go into a questionnaire you have editor access but are not the owner (using alternative account)
2. Go to the Sharing page on the global menu.
3. Click the x to remove yourself as and editor.
4. Check the red heading bar appears at the top.
